### ISSUE TITLE
Can't run any tests due to undefined log object

### ISSUE THREAD

**Issue by sslotsky** [boogie666/atom-mocha#17]
Using Atom v1.12.7, atom-mocha v0.0.8.

Just installed this very promising looking add-on, but I can't get tests to run. I get the following error no matter how I try to run the tests:

```
TypeError: Cannot read property 'error' of undefined
```

The stack trace indicates that the `log` object from the snippet below is what is undefined.

```javascript
  OptionManager.prototype.mergeOptions = function mergeOptions(opts) {
    var alias = arguments.length <= 1 || arguments[1] === undefined ? "foreign" : arguments[1];

    if (!opts) return;

    for (var key in opts) {
      if (key[0] === "_") continue;

      var option = _config2["default"][key];

      // check for an unknown option
      if (!option) this.log.error("Unknown option: " + alias + "." + key, ReferenceError);
    }

    // normalise options
    _index.normaliseOptions(opts);

    // merge them into this current files options
    _helpersMerge2["default"](this.options, opts);
  };
```

Any tips appreciated, thanks!

**Comment by bartolkaruza** [comment]
I am running into the same issue on Atom 1.13.1 and atom-mocha 0.0.8

**Comment by boogie666** [comment]
Should be fixed now can one of you try with version 0.0.9. 
It this still happens please reopen this issue. 

Cheers,
Boogie
